addons/editor-number-arrow-keys/addon.json

```diff
@@ -4,9 +4,8 @@
   "tags": ["editor", "codeEditor", "recommended"],
   "versionAdded": "1.34.0",
   "latestUpdate": {
-    "version": "1.35.0",
-    "newSettings": ["useCustom"],
-    "temporaryNotice": "You can now increment input fields that are not in blocks, like the sprite properties, and fully customize the values."
+    "version": "1.36.0",
+    "temporaryNotice": "More inputs are now supported, such as color dropdowns, and onion skinning inputs."
```

Comment:
This should probably be in a separate PR, unless it's required by the addon to work (which it probably isn't).